icoscp/station/station.py

```diff
@@ -561,42 +562,147 @@ def get(stationId):
-                             
-    sort : str, optional. The default is 'name'.
-    
+    project : str, optional. The default is 'ICOS'. If you set
+        `project` to 'all', all known stations are returned.
+
+    sort : str, optional. The default is 'name'. A user can `sort` by
+        any of the dataframe's columns: ['uri' | 'id' | 'name' |
+        'country' | 'lat' | 'lon' | 'elevation' | 'project' | 'theme']
+
+    outfmt: str, optional. The default is 'pandas'. If you provide
+        'map' to the `outfmt` argument a folium map is created with
+        all known stations that have valid longitude and latitude
+        values. Be advised that in this case, stations without a fixed
+        location (like measurements that belong to stations collected
+        from instrumented Ships of Opportunity) will not be included.
+
     Returns
     -------
-    Pandas data frame with station id's, name and country.
-    """    
-    
-    project = project.upper()
-    
-    query = sparqls.getStations()    
-    stn = RunSparql(query,'pandas').run()
+    queried_stations : pandas.Dataframe
+        `queried_stations` dataframe includes station id's, name,
+        country, and landing page (or uri) among others.
 
-    stn['project'] = stn.apply(lambda x : __project(x['uri']), axis=1)
-    stn['theme'] = stn.apply(lambda x: x['uri'].split('/')[-1].split('_')[0], axis=1)
-    
+    stations_map : folium.Map
+        `stations_map` is an interactive folium map with the available
+        stations provided by `project` argument.
+
+    Examples
+    --------
+    >>> stations_dataframe = getIdList(project='ICOS').head()
+    >>> print(stations_dataframe)
+            uri 	 id 	name 	...   elevation   project	theme
+    40 	http...  SE-Sto  Abis... 	... 	351.893      ICOS 	   ES
+    35 	http...  IT-Noe  Arca... 	... 	25.0 	     ICOS 	   ES
+    102 http...  UK-AMo  Auch... 	... 	270.0 	     ICOS 	   ES
+    48 	http...  FR-Aur  Aura...	... 	250.0 	     ICOS 	   ES
+    127 http...  1199 	 BE-F... 	... 	None 	     ICOS 	   OS
+
+    """
+
+    project = project.upper()
+    query = sparqls.getStations()
+    queried_stations = RunSparql(query, 'pandas').run()
+    queried_stations['project'] = queried_stations.apply(lambda x: __project(x['uri']), axis=1)
+    queried_stations['theme'] = queried_stations.apply(lambda x: x['uri'].
+                                                       split('/')[-1].split('_')[0], axis=1)
     if not project == 'ALL':
-        stn = stn[stn.project == project.upper()]
-    # we may have double entries....drop 
-    stn.drop
-    stn.sort_values(by=sort, inplace=True)
-    
-    return stn
+        queried_stations = queried_stations[queried_stations.project == project.upper()]
+    # Drop any existing double entries.
+    queried_stations.drop
+    # Sort queried stations by the given sort argument.
+    queried_stations.sort_values(by=sort, inplace=True)
+
+    if outfmt == 'map':
+        import requests
```

Comment:
I would prefer to have all the import statements at the top of the file. 
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
I moved all import statements to the top and grouped them according to the link you provided.
Thank you for that!